prog/dftb+/lib_elecsolvers/elsisolver.F90

```diff
@@ -455,6 +473,42 @@ subroutine TElsiSolver_init(this, inp, env, nBasisFn, nEl, iDistribFn, nSpin, iS
   end subroutine TElsiSolver_init
 
 
+  !> Checks for supported value of ELSI api, 2.6.0 release tag being the currently required version
+  !> for DFTB+ can use at the moment
+  subroutine supportedVersionNumber(major, minor, patch, dateStamp)
```

Comment:
Could you please update it to 2.6.1 being the last official release before the 20.1 release.